### PR TITLE
Initiate .gitignore file with all vim temp files and venv typical files

### PR DESCRIPTION
**Pull request by dj4ngo**
##### SUMMARY

Initiate .gitignore file with all vim temp files and venv typical files

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

gitignore
##### VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by dj4ngo** [comment]
That's the same issu than my other PR, I propose to merge it to master. Please let me know if you'd rather that I recreate a PR to develop for this one. Nexts PR will be on develop ;)

**Comment by 4383** [comment]
Please send this PR on develop branch

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#py venv
+.Python
+[Bb]in
```
Comment (AntoineCezar) [inline .gitignore]: venv should live in its own directory. So ignoring `venv` and `.venv` is ok but I won't support direct venv in the project directory.

File: {{cookiecutter.project_slug}}/.gitignore [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#py venv
+.Python
+[Bb]in
```
Comment (AntoineCezar) [inline {{cookiecutter.project_slug}}/.gitignore]: Same restriction on venv here too

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+[Ll]ib
+[Ll]ib64
+[Ll]ocal
+[Ss]cripts
+pyvenv.cfg
+venv
+.venv
+pip-selfcheck.json
+
+# Swap
```
Comment (AntoineCezar) [inline .gitignore]: A proper vim configuration avoid have all this files scattered around. Supporting every mess every editor lets behind them is not an option.

File: {{cookiecutter.project_slug}}/.gitignore [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+[Ll]ib
+[Ll]ib64
+[Ll]ocal
+[Ss]cripts
+pyvenv.cfg
+venv
+.venv
+pip-selfcheck.json
+
+# Swap
```
Comment (AntoineCezar) [inline {{cookiecutter.project_slug}}/.gitignore]: Same restriction on editors here

**Comment by AntoineCezar** [comment]
Let me know if you find my review rude or unfair. So I can get better at it.

**Comment by AntoineCezar** [comment]
Covered in a3a59a0aaecfa9ece5ced05e9592120d9591733c

**Pull request closed without merging**